charts/perses/Chart.yaml

```
@@ -3,7 +3,7 @@ name: perses
 description: Perses helm chart
 icon: https://avatars.githubusercontent.com/u/77209215?s=200&v=4
 type: application
-version: 0.4.1
+version: 0.4.2
```

```suggestion
version: 0.5.0
```

Since it's not a patch but more an improvement of the helm chart with new functionalities